server/channels/app/post.go

```diff
@@ -853,7 +865,15 @@ func (a *App) UpdatePost(rctx request.CTX, receivedUpdatedPost *model.Post, upda
 }
 
 func (a *App) publishWebsocketEventForPost(rctx request.CTX, post *model.Post, message *model.WebSocketEvent) *model.AppError {
-	postJSON, jsonErr := post.ToJSON()
+	var postJSON string
+	var jsonErr error
+	if post.Type == model.PostTypeBurnOnRead {
+		post.Message = "This message has to be revealed"
```

Comment:
Can we use translated string for this text?

Reply:
Further to our conversation @isacikgoz, this probably should be translated since we'll want to show a placeholder for notifications.